form-flow-valtimo/src/main/kotlin/com/ritense/valtimo/formflow/web/rest/FormFlowResource.kt

```diff
@@ -46,8 +46,8 @@ class FormFlowResource(
                 FormFlowStepResult(
                     stepInstance.id.id,
                     stepInstance.definition.type.name,
-                    stepInstance.definition.type.properties
-                    // TODO Include the prefilled form when this functionality is available
+                    formFlowService.getFormFlowStepTypeHandler(stepInstance.definition.type)
```

Comment:
Can also do this in 1 call, so you just say `getTypeProperties`, which will determine which handler should be used. Just not sure which one is "better"... Do you have any thoughts on this?